src/styles/Form.module.css

```diff
@@ -0,0 +1,69 @@
+:root {
+  --grey-one: #888;
+  --light: #fff;
+  --grey-dark: #e8e8e8;
+  --dark-one: #121212;
+  --blue-sky: #0290ff;
+}
+
+.formContainer {
+  width: 90%;
+  margin: 0 auto;
+}
+
+hr {
+  background: var(--grey-dark);
+  height: 3px;
+  border: none;
+}
+
+h1 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: var(--grey-one);
+  text-transform: uppercase;
+  text-align: start;
+}
+
+form {
+  display: flex;
+  column-gap: 2rem;
+}
+
+input {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid var(--grey-dark);
+  background-color: var(--light);
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: var(--dark-one);
+}
```

Comment:
- [ ] Kindly adhere 🙏🏻 to the project requirements, according to which your "Design" should match exactly with **Zeplin** design. Right now your input field's *placeholder* colour does not match the design and you need to give proper font weight` and `spacing` around it.

## Requirement
![Screenshot from 2023-08-25 19-39-00](https://github.com/kambereBr/bookstore/assets/66476329/0b10d308-79f2-4892-8ba9-ef2ab2dd4849)

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-08-26 17-02-18](https://github.com/kambereBr/bookstore/assets/66476329/2cc7de48-13a9-4c28-8606-f60dfb3e002f)


</td>
<td>

![Screenshot from 2023-08-26 17-02-04](https://github.com/kambereBr/bookstore/assets/66476329/4c935077-46a8-4e21-b71b-65ae40425cc7)

</td>
</tr>
</table>